.circleci/config.yml

```diff
@@ -16,6 +16,23 @@ jobs:
       - run: make test
       - store_test_results:
           path: junit-reports
+  coverage:
+    docker:
+      - image: circleci/node:8-browsers
+    steps:
+      - checkout
+      - run: npm config set "//registry.npmjs.org/:_authToken" $NPM_AUTH
```

Comment:
It would be nice to be able to re-use this from the `test` job, but I couldn't figure out how.